datasets/overlastgebieden/blowverbodsgebied/v1.0.0.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "blowverbodsgebied",
+  "title": "Blowverbodsgebied",
+  "type": "table",
+  "version": "1.0.0",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+      "schema",
+      "id"
+    ],
+    "display": "type",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "id": {
+        "type": "integer",
+        "description": "Unieke aanduiding van het overlastgebied."
+      },
+      "geometry": {
+        "$ref": "https://geojson.org/schema/MultiPolygon.json",
+        "description": "Polygoon definitie van het overlastgebied."
+      },
+      "type": {
+        "type": "string",
+        "description": "Code die het type overlastgebied aangeeft: (BLV) blowverbodsgebied."
+      },
+      "typering": {
+        "type": "string",
+        "description": "Omschrijving van de code die het type overlastgebied aangeeft: (BLV) blowverbodsgebied."
+      },
```

Comment:
Het is niet onmiddellijk duidelijk wat dit betekent. Is de waarde van beide velden altijd `(BLV) blowverbodsgebied`?

Reply:
Hoi, 
- type is de waarde idd BLV... in het bron bestand komen meerdere verschillende type voor... 
- Typering is een omschrijving van TYPE.. Dit heb het format overgenomen vanuit andere beschrijvingen..

Reply:
gezien het aantal de zelfde tabellen geef ik je gelijk! Het had in 1 tabel moeten zijn met daarin alle oov records.. Helaas was dit zo toen ik hier begon.. Dank voor het goedkeuren en mergen van het pull request!